features/organisations.feature

```diff
@@ -17,7 +17,7 @@ Feature: List organisations
 
   Scenario: Visit the list page
     When I visit the home page
-    Then I should see "Stub User"
+    Then I should see "@example.com"
```

Comment:
Can we test that we see the user's exact email?

Reply:
I've added this as a separate commit. I agree it's a good one to do but it means we've diverged from the upstream again so I'm a lil bit 😬  at maybe adding another thing to manage. Not sure if this is really an issue tho!